### ISSUE TITLE
New design for comments

### ISSUE THREAD

**Issue by inyono** [serlo-org/athene2#318]

**Comment by Knorrke** [comment]
I don't think this is the task we discussed, should we change the description? The changes we discussed:
* Always show field for comment, not collapsed (when logged in)
* Remove info-text
* Redesign textarea
* Remove discussion-title
* Rename "Discussions" to "Comments"

I made these changes and will create the PR as soon as this is clarified.
